Sources/SwiftGenKit/Parsers/Strings/StringsParser.swift

```diff
@@ -9,21 +9,28 @@ import PathKit
 
 public enum Strings {
   public enum ParserError: Error, CustomStringConvertible {
-    case duplicateTable(name: String)
+    case duplicateKeysInTable(name: String, extensions: [String], keys: [String])
     case failureOnLoading(path: String)
     case invalidFormat
     case invalidPlaceholder(previous: Strings.PlaceholderType, new: Strings.PlaceholderType)
+    case unsupportedFileType(path: Path, supported: [String])
 
     public var description: String {
       switch self {
-      case .duplicateTable(let name):
-        return "Table \"\(name)\" already loaded, cannot add it again"
+      case .duplicateKeysInTable(let name, let extensions, let keys):
+        // swiftlint:disable:next line_length
+        return "Table \"\(name)\" already loaded by other parser, the parser for [\(extensions.map({ ".\($0)" }).joined(separator: ", "))] files cannot modify existing keys:\n\(keys.joined(separator: "\n"))"
```

Comment:
nit: maybe reformat that in multiple lines (line we did with `.unsupportedFileType` below) to both make that nicer to read and avoid `line_length` violation?